docs/decisions/0005-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,19 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to staff users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django service. This method of granting permissions is not endorsed, since it is difficult to review, audit, and track changes to user access over time.
+
+edx-platform, however, defines ``manage_user`` and ``manage_group`` management commands, which allow users to be managed via an external system (such as one, for example, that defines permissions declaratively in version-controlled YAML). These user and group management commands are supposedly generic enough to be used via a common library/app (such as edx-django-utils) so that this users and group management scheme can be brought to other IDAs.
+
+Decision
+--------
+
+User and group management commands should be moved to edx-django-utils from edx-platform and ``edx_django_utils.user`` should be added to ``INSTALLED_APPS`` section of all django services so that the user and group management commands are available to the services.
```

Comment:
Whether you choose to duplicate or move part of this, the installed apps help text should be in the user README.

Reply:
Done.